Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesStrings.uni

```diff
@@ -77,3 +77,9 @@
 
 #string STR_FAN_CURVE_SILENT        #language en-US "Silent"
 #string STR_FAN_CURVE_PERFORMANCE   #language en-US "Performance"
+
+#string STR_DMA_BOOT_ENABLE_PROMPT      #language en-US "DMA Protection"
+#string STR_DMA_BOOT_ENABLE_HELP        #language en-US "Enables DMA protection"
+
+#string STR_IOMMU_HANDOFF_ENABLE_PROMPT      #language en-US "IOMMU handoff at ExitBootServices"
+#string STR_IOMMU_HANDOFF_ENABLE_HELP        #language en-US "Enable IOMMU when transfer control to OS (ExitBootService in normal boot. EndOfPEI in S3)"

```

Comment:
This line needs information what happens if the handoff is nto done at EBS. Also we do not support S3 in UefiPayloadPkg, S3 resume happens in coreboot already.

Reply:
Fixed.